i18n/de.json

```diff
@@ -22,13 +22,19 @@
   "status": "Status",
   "creation time": "Erstelldatum",
   "topic details": "Details",
-  "no subtopic": "Für diesen Inhalt existieren keine Unterthemen",
+  "no subtopic": "Für diesen Inhalt existieren keine Unterthemen.",
+  "parent topics": "Übergeordneten Themen",
+  "Topic updated" : "Thema aktualisiert",
+  "Status updated": "Status aktualisiert",
 
   "email placeholder": "name@provider.de",
   "first name": "Vorname",
```

Comment:
Please use `first name`, `last name`, `role` in all occurences - then we'll just need one translation.